rust/examples/datadog.rs

```diff
@@ -13,7 +13,7 @@ pub async fn main() -> anyhow::Result<()> {
     let engine = wasmtime::Engine::new(&config)?;
     let module = wasmtime::Module::new(&engine, &data)?;
 
-    let ddconfig = DatadogConfig::new();
+    let ddconfig = DatadogConfigBuilder::default().build()?;
```

Comment:
could maybe get the default from DatadogConfig if you aren't gonna set anything. will have some examples of stuff you can set here though